samples/certs/generate-workload.sh

```diff
@@ -15,17 +15,26 @@
 #   limitations under the License.
 
 name=${1:-foo}
-san="spiffe://trust-domain-$name/ns/$name/sa/$name"
+ns=${2:-$name}
+sa=${3:-$name}
+dir=${4:-"."}
+tmp=${5:-""}
```

Comment:
why we need both dir and tmp? can we just use one directory?

Reply:
the tmp is for the testing certs, since it uses the foo and bar, which would overwrite the existing certs.

Reply:
you can use tmp=$(mktmp). This is used more common in automation.

Also currently you are exposing 5 positional arguments to this shell script. It's unlikely a dev to memorize and actual fill 5 value sin the CLI.

Just assign name ot ns/sa. dir you can get it via some bash onlines, like `$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )`.  See other shell for e.g.

Reply:
Ignore the `tmp` one i realized you need that as argument to invoke from golang code. other arguments i think can still be reduced.

Reply:
removed dir as argument